test-data/unit/semanal-classes.test

```diff
@@ -252,7 +252,7 @@ MypyFile:1(
       IntExpr(1))
     AssignmentStmt:3(
       NameExpr(y* [m])
-      NameExpr(x [m]))))
+      NameExpr(x [__main__.A.x]))))
```

Comment:
This is because of the mentioned change in ``semanal.py``. Previously ``fullname`` was ``None`` in these examples, so that they were formatted in simplified manner, now they show their ``fullname``s.